api/server.go

```diff
@@ -34,9 +35,26 @@ func (d *dispatcher) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 
 // Server is a Docker API server.
 type Server struct {
-	hosts      []string
-	tlsConfig  *tls.Config
-	dispatcher *dispatcher
+	cfg          *Config
```

Comment:
I see this is where the `Config` struct below on L#50 is used, but where is this pointer used?